go/vt/sqlannotation/sqlannotation.go

```diff
@@ -25,6 +27,10 @@ const (
 	deleteDML                            = "delete"
 )
 
+var (
+	replicationUnfriendlyCountStat = stats.NewInt("replication-unfriendly-statements-count")
```

Comment:
I just saw this review :)
Variable names have to be camel cased. Also, /debug/vars produces JSON, which means that you can have hierarchical values. They get translated correctly on import.